Docs/source/install/dependencies.rst

```diff
@@ -119,16 +119,24 @@ Without MPI:
 .. code-block:: bash
 
    conda create -n warpx-dev -c conda-forge blaspp ccache cmake compilers git lapackpp openpmd-api python numpy scipy yt fftw matplotlib mamba ninja
-   conda activate warpx-dev
+   source activate warpx-dev
 
    # compile WarpX with -DWarpX_MPI=OFF
 
 With MPI (only Linux/macOS):
 
 .. code-block:: bash
 
-   conda create -n warpx-dev -c conda-forge blaspp ccache cmake compilers git lapackpp openpmd-api=*=mpi_openmpi* python numpy scipy yt fftw=*=mpi_openmpi* matplotlib mamba ninja openmpi
-   conda activate warpx-dev
+   conda create -n warpx-dev -c conda-forge blaspp ccache cmake compilers git lapackpp "openpmd-api=*=mpi_openmpi*" python numpy scipy yt "fftw=*=mpi_openmpi*" matplotlib mamba ninja openmpi
```

Comment:
Quotes added because `zsh` on macOS otherwise had troubles escaping it. Compatible with Linux/Windows use.